release.nix

```diff
@@ -10,11 +10,11 @@ let
   in {
     recurseForDerivations = true;
     app = example.hsPkgs.reflex-todomvc.components.exes.reflex-todomvc;
-    ghcShell = example.shells.ghc;
+    ghcjs-app = example.ghcjs-app;
     inherit (tests) testOut;
   } // pkgs.lib.optionalAttrs (system == "x86_64-linux") {
     android-app = example.android.app.aarch64;
-    ghcjs-app = example.ghcjs-app;
+    ghcShell = example.shells.ghc; # TODO: triggers unintended eval of 'webkitgtk' which is broken on darwin
```

Comment:
https://github.com/reflex-frp/reflex-platform/commit/05907c12d84f45942b0b5b7a1ef00d090ef2d239 seems like it *meant* to do this but moved the ghcjs-app instead?

Reply:
Yeah that seems to be the case, I may have also mislabeled the commit.. either way this change is fine!